manifests/osfamily/darwin.pp

```diff
@@ -1,14 +1,6 @@
 class puppet_agent::osfamily::darwin{
```

Comment:
What do the last two lines of the commit message mean?

Reply:
Oh, those are garbage. Must have been a copy/paste error in my editor